File: Demos/Datasnap/Server/proxy/objectivec_ios81/Proxy/DSAdminRestClient.m
//*******************************************************
//
//               Delphi DataSnap Framework
//
// Copyright(c) 1995-2011 Embarcadero Technologies, Inc.
//
//*******************************************************

#import "DSAdminRestClient.h"

@implementation DSAdminRestClient

@synthesize Connection;

-(id) initWithConnection:(DSRESTConnection *) aConnection {
	self  = [self init];
	if (self) {
		self.Connection = aConnection ;
	}
	return self;
	
}
@end

File: Demos/Datasnap/Server/proxy/java_blackberry/com/embarcadero/javablackberry/DSAdminRestClient.java
//*******************************************************
//
//               Delphi DataSnap Framework
//
// Copyright(c) 1995-2011 Embarcadero Technologies, Inc.
//
//*******************************************************

package com.embarcadero.javablackberry;


public class DSAdminRestClient {

	private DSRESTConnection Connection = null;

	protected DSRESTConnection getConnection() {
		return Connection;
	}

	public DSAdminRestClient(DSRESTConnection Connection) {
		super();
		this.Connection = Connection;
	}

}
